pyproject.toml

```diff
@@ -10,7 +10,7 @@ dependencies = [
     "jsonschema",
     "lightkube",
     "lightkube-models",
-    "ops",
+    "ops[tracing]~=2.21",
```

Comment:
Do we need to freeze the dependency here? We use uv to achieve this

Reply:
note that I'm using the "compatible" operator, `~=2.21` means `>=2.21.0,<3.0.0`.
2.21.0 is the first version that has this optional dependency group.